.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,45 @@
+
+name: deploy docs
+on:
+  release:
+    types: [published]
+    
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: install prerequisites
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+          libdrm-dev \
+          libegl1-mesa-dev \
+          libgl1-mesa-dev \
+          libx11-dev \
+          libxext-dev \
+          libxfixes-dev \
+          libwayland-dev \
+          libx11-xcb-dev \
+           libxcb-dri3-0 \
+          doxygen 
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.10' 
+
+    - name: Build
+      run: |
+        python -m pip install ninja
+        python -m pip install meson
+        meson _build -Denable_docs=true
+        meson compile -C _build
+
+    - name: Build echo
+      run:  echo ${{ github.ref }}
```

Comment:
This is a no-op - drop?

Reply:
It was removed

Reply:
The change should be squashed appropriately